lib/assertion.js

```diff
@@ -169,6 +180,20 @@ class Assertion extends TestResultReporter {
   }
   
   // Private
+
+  _identityAssertion(expected, shouldBeIdentical) {
+    const { comparisonResult, overrideFailureMessage } =
+      IdentityAssertionStrategy.evaluate(this._actual, expected);
+    const resultIsSuccessful = shouldBeIdentical ? comparisonResult : !comparisonResult;
+
+    if (isUndefined(comparisonResult) || overrideFailureMessage) {
```

Comment:
could this be simplified to just `isUndefined(comparisonResult)`? looking at the identity rule objects, having a result undefined implies having an `overrideFailureMessage`

Reply:
Thank you for the great feedback! 😄

That's a good one! I thought about this and couldn't decide on one of the two conditions, but should have 😰. Also thought that having an `overrideFailureMessage` defined would mean that the result of the evaluation has not been successful. I'll leave just the `isUndefined(comparisonResult)` as you suggested!

Maybe, in the future it would be useful that the `evaluate` message returned a reification of the comparison. Just thinking out loud... it may let us simplify this method 🤔.